include/AudioSdl.h

```diff
@@ -62,7 +62,7 @@ class AudioSdl : public AudioDevice
 		setupWidget( QWidget * _parent );
 		virtual ~setupWidget();
```

Comment:
If this dtor is overriding, it should be marked `override` as well (some compilers complain if not all functions are marked `override`).